modules/core/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -390,4 +390,44 @@ public static class NodeConfiguration {
          */
         public static final int CONFIG_WRITE_ERR = NODE_CONFIGURATION_ERR_GROUP.registerErrorCode(3);
     }
+
+    /**
+     * Code deployment error group.
+     */
+    public static class CodeDeployment {
+        /**
+         * Code deployment error group.
+         */
+        public static final ErrorGroup CODE_DEPLOYMENT_ERR_GROUP = ErrorGroup.newGroup("CODEDEPLOY", 13);
+
+        /**
+         * Config read error.
+         */
+        public static final int UNDEPLOY_NOT_EXISTED_ERR = CODE_DEPLOYMENT_ERR_GROUP.registerErrorCode(1);
```

Comment:
```suggestion
        public static final int UNDEPLOY_NON_EXISTING_UNIT_ERR = CODE_DEPLOYMENT_ERR_GROUP.registerErrorCode(1);
```

"Not existed" is not valid English (except "has not existed", but that's not what we want here).

Reply:
Can we remove this error and use `UNIT_NOT_FOUND_ERR` below when someone tries to undeploy a unit that does not exist?